autogen

```diff
@@ -161,6 +162,13 @@ Options:
                            add a top-level documentation comment after the
                            license header (default: enabled)
 
+  --no-runline, --no-rnl
```

Comment:
I would suggest dropping the `--no-rnl` form: it is not a easily-recognized/remembered abbreviation, it's not a first-letter-of-each-of-the-words, so it doesn't add much. If it's ever used in a script, it won't be clear to anyone what it means without looking at the docs, so it's not self-documenting.

I'll admit the `--no-tlc` was a bit of a fun pun for me as much as it was a first-letter-of-each-of-the-words shortening of `--no-top-level-comment`. :-)